honeybee_energy/programtype.py

```diff
@@ -5,34 +5,38 @@
 
 
 class ProgramType(object):
-    """Energy Program Type.
+    """Program Type object possessing all schedules and loads defining a program.
 
     Properties:
         name
-        people_load
-        lighting_load
-        electric_equipment_load
-        gas_equipment_load
-        infiltration_load
-        ventilation_load
+        people
+        lighting
+        electric_equipment
+        gas_equipment
+        infiltration
+        ventilation
+        cooling_setpoint_schedule
+        heating_setpoint_schedule
     """
 
-    def __init__(self, name, people_load=None, lighting_load=None,
-                 electric_equipment_load=None, gas_equipment_load=None,
-                 infiltration_load=None, ventilation_load=None):
+    def __init__(self, name, people=None, lighting=None, electric_equipment=None,
+                 gas_equipment=None, infiltration=None, ventilation=None,
+                 cooling_setpoint_schedule=None, heating_setpoint_schedule=None):
         """Initialize energy program type.
 
         Args:
             name: Text string for construction program type. Must be <= 100 characters.
                 Can include spaces but special characters will be stripped out.
         """
         self.name = name
-        self.people_load = people_load
-        self.lighting_load = lighting_load
-        self.electric_equipment_load = electric_equipment_load
-        self.gas_equipment_load = gas_equipment_load
-        self.infiltration_load = infiltration_load
-        self.ventilation_load = ventilation_load
+        self.people = people
```

Comment:
We will definitely have checks.  Either the next PR or the one after it will include them here.  I'm just trying to break these major feature PRs down into bite-size chunks for review without breaking the tests so that is why there's still a bit of paceholder code here.